config/environments/development.rb

```diff
@@ -31,6 +31,8 @@
   # Raise an error on page load if there are pending migrations.
   # config.active_record.migration_error = :page_load
 
+  config.cache_store = :redis_cache_store, { url: Settings.redis.app_data.url, expires_in: 30.minutes }
```

Comment:
I totally picked 30.minutes arbitrarily, but it seemed like a good start since it's the default session length.  I'm assuming that most will have an explicit override for each kind of cache.